docs/installation_guide.md

```diff
@@ -100,19 +100,24 @@ The parameters are described below - note that at the very least, you will need
 
 - **harbor_admin_password**: The administrator's initial password. This password only takes effect for the first time Harbor launches. After that, this setting is ignored and the administrator's password should be set in the Portal. _Note that the default username/password are **admin/Harbor12345** ._
 
-
-
 - **database**: the configs related to local database
-  - **password**: The root password for the PostgreSQL database used for **db_auth**. _Change this password for any production use!_
+  - **password**: The root password for the PostgreSQL database. Change this password for any production use.
+  - **max_idle_conns**: The maximum number of connections in the idle connection pool. If <=0 no idle connections are retained. The default value is 2.
```

Comment:
**max_idle_conns**: The maximum number of connections in the idle connection pool. If <=0 no idle connections are retained. Note that the default value is 50 and if it is not configured the value is 2.

Reply:
Fixed in https://github.com/goharbor/harbor/pull/9005/commits/0afffd027ee83f120e31ed7dd7067a91d27d89a0.